pika/adapters/blocking_connection.py

```diff
@@ -1541,30 +1541,30 @@ def basic_consume(self,
             consumer_tag is already present.
 
         """
-        if not callable(callback):
-            raise ValueError('callback callback must be callable; got %r'
+        if not callable(on_message_callback):
+            raise ValueError('callback on_message_callback must be callable; got %r'
                              % callback)
```

Comment:
Per pylint: "Undefined variable 'callback'". Sounds like we are missing a unit test for `not callable(on_message_callback)`.